.github/workflows/build.yml

```diff
@@ -479,6 +486,13 @@ jobs:
           bash --version
           echo "boundary version:"
           boundary version
+          echo "vault version:"
+          vault version
       - name: Run cli bats tests
         run: |
+          export VAULT_ADDR=http://127.0.0.1:8200
+          export VAULT_TOKEN=boundarytok
+          make -C internal/tests/cli test-vault-up
+          until vault status; do docker container inspect boundary-cli-tests-vault &> /dev/null || exit -1; sleep 1; done
           make test-cli
+          make -C internal/tests/cli test-vault-down
```

Comment:
Does this run if a bats test fails?

Reply:
Split this out into a separate step that will run even on failure.